archinstall/default_profiles/desktops/xmonad.py

```diff
@@ -0,0 +1,27 @@
+from typing import override
+
+from archinstall.default_profiles.profile import GreeterType, ProfileType
+from archinstall.default_profiles.xorg import XorgProfile
+
+
+class xmonadProfile(XorgProfile):
+	def __init__(self) -> None:
+		super().__init__('Xmonad', ProfileType.WindowMgr, description='')
+
+	@property
+	@override
+	def packages(self) -> list[str]:
+		return [
+			'xmonad',
+			'xmonad-contrib',
+			'xmonad-extra',
+			'xterm',
+			'lightdm-gtk-greeter',
```

Comment:
this shouldn't be installed explicitly as it's dynamically added based on the greeter